lib/pbench/test/unit/server/database/test_datasets_metadata_db.py

```diff
@@ -189,42 +189,44 @@ def test_user_metadata(self, attach_dataset):
         # See if we can create a metadata row
         ds = Dataset.query(name="drb")
         user1 = User.query(username="drb")
+        metadata_db_query = Database.db_session.query(Metadata)
         assert ds.metadatas == []
-        t = Metadata.create(key="user", value=True, dataset=ds, user=user1)
+        t = Metadata.create(key="user", value=True, dataset=ds, user_id=user1.id)
         assert t is not None
         assert ds.metadatas == [t]
-        assert user1.dataset_metadata == [t]
+        assert Database.db_session.query(Metadata).filter_by(
+            user_id=user1.id
+        ).all() == [t]
 
         user2 = User.query(username="test")
-        d = Metadata.create(key="user", value=False, dataset=ds, user=user2)
+        d = Metadata.create(key="user", value=False, dataset=ds, user_id=user2.id)
         assert d is not None
         assert ds.metadatas == [t, d]
-        assert user1.dataset_metadata == [t]
-        assert user2.dataset_metadata == [d]
```

Comment:
Why did you drop the assertion at (old) line 202?  It seems to me that it's still a worthwhile thing to check (and you check it after the subsequent operations...).

Reply:
yep, I missed adding it back